java/src/jmri/jmrit/logixng/LogixNGBundle.properties

```diff
@@ -113,3 +113,6 @@ AddressByMemory_Listen          = by memory "{0}" ({1})
 AddressByLocalVariable          = by local variable "{0}"
 AddressByFormula                = by formula "{0}"
 AddressByTable                  = by table {0}, row {1}, column {2}
+
+CsvType_Tabbed          = Tabbed
+CsvType_Comma           = Comma
```

Comment:
I think these two lines should be removed.
Please ensure that the last line in the file is an empty line.

Reply:
They are used in the Table Interface. Added a blank line after them